### PR TITLE
New billing style.

### PR DESCRIPTION
**Pull request by MiguelVis**
New billing style for Daloradius.

Its use it's optional, and one can still using the traditional style or change to the new style, or revert, at any time.

It lets you use a lot of new fields for the design and cutomization of your invoices.

It use two HTML templates: one for the invoice body, and another for the invoice items.

invoice_preview2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 .idea/
 *.log
+*.db
+invoice_preview.html
```
Comment (lirantal) [inline .gitignore]: this is an automatically generated file? is this why we're ignoring it?

**Comment by lirantal** [comment]
Nice stuff @MiguelVis!

File: .gitignore (hunk 1 above)
Comment (MiguelVis) [reply .gitignore]: Right.

Comment (lirantal) [reply .gitignore]: Cool thanks for clarifying.

**Pull request merged**